File: quark/test/emit/booleans/py/booleans.py
from quark_runtime import *


def main():
    if (True):
        _println(u"Hi!");

    b = (1) > (0);
    if (b):
        _println(u"Hey!");

    c = False;
    if (not (c)):
        _println(u"Ho!");

    count = 0;
    while (True) :
        if ((count) > (3)):
            return

        count = (count) + (1)



if __name__ == "__main__":
    main()

File: quark/test/emit/booleans/js/booleans.js
var _qrt = require("quark_runtime.js");

function main() {
    if (true) {
        _qrt.print("Hi!");
    }
    var b = (1) > (0);
    if (b) {
        _qrt.print("Hey!");
    }
    var c = false;
    if (!(c)) {
        _qrt.print("Ho!");
    }
    var count = 0;
    while (true) {
        if ((count) > (3)) {
            return;
        }
        count = (count) + (1);
    }
}
exports.main = main;

main();
